front/app/hooks/useAdminPublications.ts

```diff
@@ -77,26 +77,12 @@ export default function useAdminPublications({
   const [areas, setAreas] = useState<string[] | undefined>(areaFilter);
   const [publicationStatuses, setPublicationStatuses] = useState<
     PublicationStatus[]
-  >(publicationStatusFilter);
+  >([]);
 
-  const onLoadMore = useCallback(() => {
-    if (hasMore) {
-      setLoadingMore(true);
-      setPageNumber((prevPageNumber) => prevPageNumber + 1);
-    }
-  }, [hasMore]);
-
-  const onChangeAreas = useCallback((areas) => {
-    setAreas(areas);
-    setPageNumber(1);
-  }, []);
-
-  const onChangePublicationStatus = useCallback((publicationStatuses) => {
-    setPublicationStatuses(publicationStatuses);
-    setPageNumber(1);
-  }, []);
+  useEffect(() => {
+    setPublicationStatuses(publicationStatusFilters);
+  }, [publicationStatusFilters]);
```

Comment:
I think this can be solved by memoizing (useMemo) the publicationStatusFilters in useAdminPublications, use this memoized array as the dependency here in the useEffect/to set the state and then you don't have to worry about it anywhere else. You can have a look in the article I linked to in the other PR: https://kentcdodds.com/blog/usememo-and-usecallback.

Reply:
Looks like a textbook useMemo use case, but to be tested :-)